app/models/item.rb

```diff
@@ -11,6 +14,6 @@ class Item < ApplicationRecord
     }
     
     def self.search(query)
-        Item.all.where("lower(name) LIKE ? OR lower(category) LIKE ? OR lower(description) LIKE ?","%"+query+"%","%"+query+"%","%"+query+"%")
+        Item.all.where("lower(name) LIKE ? OR category = ? OR lower(description) LIKE ?","%"+query+"%",Item.categories[query.capitalize],"%"+query+"%")
```

Comment:
is this sql injection safe?

Reply:
this isn't really injecting anything into the database. i think this should work.